packages/vite-plugin-svelte/package.json

```
@@ -23,7 +21,7 @@
     "generate:types": "dts-buddy -m \"@sveltejs/vite-plugin-svelte:src/public.d.ts\""
   },
   "engines": {
-    "node": "^18.0.0 || ^20.0.0 || >=22"
+    "node": "^20.19 || ^22.12 || >=24"
```

```suggestion
    "node": "^20.17 || ^22 || >=24"
```
Same as https://github.com/sveltejs/vite-plugin-svelte/pull/1129/files#r2127851453